lib/mjsonwp/errors.js

```diff
@@ -300,6 +363,17 @@ class NotImplementedError extends MJSONWPError {
   }
 }
 
+class UnableToCaptureScreen extends MJSONWPError {
+  static w3cStatus () {
+    return HTTPStatusCodes.INTERNAL_SERVER_ERROR;
+  }
+  constructor (err) {
+    super(err || 'Unable to capture screen', null, UnableToCaptureScreen.w3cStatus());
```

Comment:
is  it ok to set jsonwp code to null?

Reply:
I'll set it to 13 by default if null is provided. 13 is just an `Unknown Error`